tests/e2e/e2e_test.go

```diff
@@ -60,7 +60,7 @@ func (s *IntegrationTestSuite) Test02CreatePool() {
 // - voting yes on the proposal from the validator wallet
 // - voting no on the proposal from the delegator wallet
 // - ensuring that delegator's wallet overwrites the validator's vote
-func (s *IntegrationTestSuite) TestSuperfluidVoting() {
+func (s *IntegrationTestSuite) Test03SuperfluidVoting() {
```

Comment:
Superfluid Voting should no longer be required to be ran in a specific order

Reply:
We should order these to be run after pool creation. Otherwise, it does not make sense having that test. If pool creation fails, we cannot rn any other tests since they all create pools

Reply:
Err, you can't rely on test ordering being lexicographic ordering. But I also don't think we should be have test ordering based on naming, and should be specified elsewhere

Reply:
We don't rely on this order. If the order was changed, the tests would still run because they are independent. 

The reason for this lexicographic naming is to be able to isolate 'TesteCreatePool' conveniently and run it first because we create pools in the majority of other tests.

If 'TestCreatePool' were faulty, we would cancel the remaining tests.

If we don't have 'TestCreatePool' run first, I think we might not need a dedicated fixture for testing pool creation because it would be covered by other flows.

Let me know what you think. If no specific opinions, I can remove the lexicographic order and consider removing 'TestCreatePool'

Reply:
Ordering is removed